File: Python/BOJ_2941_크로아티아 알파벳.py
import sys

string = sys.stdin.readline().rstrip()
croatia = ['c=', 'c-', 'dz=', 'd-', 'lj', 'nj', 's=', 'z=']

for i in croatia:
    string = string.replace(i, '@')
print(len(string))

File: C++/BOJ_2941_크로아티아 알파벳.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);

    int index;
    string str;
    vector<string> croatia = { "c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z=" };
    
    cin >> str;
    for (int i = 0; i < croatia.size(); i++) {
        while(1) {
            index = str.find(croatia[i]);
            if (index == string::npos) break;
            str.replace(index, croatia[i].length(), "@");
        }
    }
    cout << str.length();

    return 0;
}
